pkg/controllers/node/node_controller.go

```diff
@@ -79,6 +80,11 @@ func NewNodeController(ctx context.Context, k8sClientset *kubernetes.Clientset,
 		nodeCache:    make(map[string]*api.Node),
 	}
 
+	// Create a default profile for auto hostendpoints
+	err := nc.createAllowProfile()
+	if err != nil {
+		os.Exit(1)
```

Comment:
I think this means that if the KubeAPIServer is not available, or we don't have permission, we'll just crash in a loop.